Eatery/Views/Eateries/HoursThisWeekView.swift

```diff
@@ -0,0 +1,245 @@
+//
+//  HoursThisWeekView.swift
+//  Eatery
+//
+//  Created by William Ma on 2/4/20.
+//  Copyright © 2020 CUAppDev. All rights reserved.
+//
+
+import SnapKit
+import UIKit
+
+class HoursThisWeekView: UIView {
+
+    // only shown for dining halls
+    private let segmentedControl = UISegmentedControl()
+    private var showDiningHallConstraints: [Constraint] = []
+    private var hideDiningHallConstraints: [Constraint] = []
+
+    private let verticalSeparator = UIView()
+    private let stackView = UIStackView()
+
+    private var eatery: Eatery?
+
+    private let diningMeals = ["Breakfast", "Lunch", "Dinner"]
+
+    init() {
+        super.init(frame: .zero)
+
+        segmentedControl.addTarget(self,
+                                   action: #selector(segmentedControlValueChanged(_:)),
+                                   for: .valueChanged)
+        addSubview(segmentedControl)
```

Comment:
Is this how we do line wrapping? Pretty sure the Google style guide [says](https://google.github.io/swift/#function-calls) to do it like this:
```
segmentedControl.addTarget(
    self,
    action: #selector(...),
    for: .valueChanged
)
```
Also, [Tactile](https://github.com/delba/Tactile) provides a really nice condensed syntax (and support for closures!) if you're interested.